README.md

```diff
@@ -214,23 +203,15 @@ GitHub: [Alice Mirigo](https://github.com/alicemirigo92)
 - Twitter: [@Gathonimirigo](https://twitter.com/GathoniMirigo)
 - LinkedIn: [Alice](https://www.linkedin.com/in/alice-mirigo/)
 
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- FUTURE FEATURES -->
 
 ## 🔭 Future Features <a name="future-features"></a>
 
-> Describe 1 - 3 features you will add to the project.
-
-- [ ] **[new_feature_1]**
-- [ ] **[new_feature_2]**
-- [ ] **[new_feature_3]**
+- [ ] **[Add more advanced tables]**
```

Comment:
- Please be sure to remove the template text from you README to make it more professional. 
![Screenshot_20230405-131011_Chrome](https://user-images.githubusercontent.com/44624138/230076911-1b72a562-f032-4b1f-a8b0-06f4185f53b3.jpg)

Reply:
![Screenshot_20230405_131104](https://user-images.githubusercontent.com/44624138/230077604-60adbce0-96ff-478e-b3b3-f39e2a498991.jpg)
This project project do not include a live demo, so it will be nice to remove the section or add not available 😉.